### PR TITLE
include kwargs to parse to scipy in to support the new parameter.

### PR DESCRIPTION
**Pull request by ahms5**
- tests are failing atm
- remove attributes for init, since init is not supposed the get called directly.
- add **kwargs to make it wok with [scipy's](https://github.com/scipy/scipy/releases/tag/v1.14.0) new parameter scalar-first (it is also backwardcompatible)
- change from ``'Hz'`` parameter in [scipy](https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.signal.remez.html)

also general remark: we can discuss if its a good idea to overwrite the init function, just to allow to run 'pf.Orientations()' even if the orientation is not empty, beacuse scipy does not allow it.

### CODE REVIEW

**Comment by f-brinkmann** [comment]
> also general remark: we can discuss if its a good idea to overwrite the init function, just to allow to run 'pf.Orientations()' even if the orientation is not empty, beacuse scipy does not allow it.

Is this required to for `Orientations.from_view_up(views, ups)` to work?

**Comment by f-brinkmann** [comment]
Regarding https://github.com/pyfar/pyfar/pull/638/commits/89b4977685d61006be297e901826e65173a34b21: Might it be that it only has to come before the Examples section?

**Comment by ahms5** [comment]
> before

I think its just confusing for the user if we document the init paramers, even they should not use it. so thats why i was thinking to remove it

**Comment by ahms5** [comment]
> > also general remark: we can discuss if its a good idea to overwrite the init function, just to allow to run 'pf.Orientations()' even if the orientation is not empty, beacuse scipy does not allow it.
> 
> Is this required to for `Orientations.from_view_up(views, ups)` to work?

I treid it, it was working. It works because its calling the init from scipy. the only thing which is not working anymore is pf.Orientation() beause scipy does not allow it, the ``'quat'`` parameter need to be set .

**Comment by f-brinkmann** [comment]
> I tried it, it was working.

I'd be ok with removing it in this case. But maybe up for discussion. It would break backwards compatability if anyone ever used `pf.Orientations()`

> I think its just confusing for the user if we document the init paramers

Maybe this is better to be part of #631?

**Comment by ahms5** [comment]
I reverted the change pf the attributes doc, we can discuss it later on, but anyway, you are right, its not part of this pr

File: pyfar/classes/orientations.py [hunk 1]
```diff
@@ -58,10 +58,10 @@ class Orientations(Rotation):
 
     """
 
-    def __init__(self, quat=None, normalize=True, copy=True):
+    def __init__(self, quat=None, normalize=True, copy=True, **kwargs):
```
[the remark this answers is not in the document]
Comment (ahms5) [reply pyfar/classes/orientations.py:61]:
```suggestion
    def __init__(self, quat=None, **kwargs):
```
I think you are right, I would be fine with that too, but I thought it was beyond the hotfix.

[the remark this answers is not in the document]
Comment (pingelit) [reply pyfar/classes/orientations.py:61]: Yes, that might be a change for the PR of #639. I will approve for now.

**Review by pingelit — APPROVED** [review]

**Review by f-brinkmann — APPROVED** [review]
Thanks for fixing

**Pull request merged**